src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -82,16 +82,19 @@ export const ArtworkSidebarPartnerInfo: FC<ArtworkSidebarPartnerInfoProps> = ({
           )}
         </PartnerContainer>
 
-        {isCBNEnabled && !isInquireable && !is_in_auction && (
```

Comment:
Interesting that the `isInquireable` check was not sufficient for hiding the contact button 🤔 

I believe that maps back to this in Gravity:

https://github.com/artsy/gravity/blob/ffd6b8cb3e17d0258c732d88a4a47d71956e3bbe/app/models/domain/artwork.rb#L666-L671

```ruby
  def inquireable?
    return false if offer || ecommerce
    return true if partner.try(:inquireable?)

    !!(forsale? || at_auction?)
  end
```

The partner in question:

```ruby
Partner.find('walters-art-museum').inquireable?
# => false
```

Reply:
Oh wait, that check is a negation? `!isInquireable`

Reply:
Yeah, it's definitely convoluted at the moment. I think the intent was to show this button only if we're not exposing Contact Gallery as a primary commercial CTA.